docs/book/spell-check-custom-words.txt

```diff
@@ -188,4 +188,18 @@ tuple's
 unary
 SRC
 DEX
-SubId
\ No newline at end of file
+SubId
+Pausable
+Libs
+Reentrancy
+reentrancy
+mathematic
+Soulbound
+NFTs
+NFT
+dApps
+fungible
+FizzBuzz
```

Comment:
Instead of adding non-words to this list, please wrap them in backticks like `FizzBuzz` so the spell checker will ignore them. [Here](https://github.com/FuelLabs/github-actions/blob/master/docs-hub/README.md#handling-spell-check-errors) is a reference for how to handle spelling errors in general.